### PR TITLE
Empty graves can now be spawned

### PR DESCRIPTION
**Pull request by ishitbyabullet**
## About The Pull Request

Graves can now spawn empty, allowing mappers (or maybe even admins/players) to not worry about it having items they didn't intend to have in there.

Fikou gave me the idea to make this PR.

## Why It's Good For The Game

I just think it would be neat for empty versions of graves to exist.

## Changelog
:cl: Baldies
qol: empty versions of graves can now be spawned in.
/:cl:

### CODE REVIEW

File: code/modules/ruins/lavalandruin_code/elephantgraveyard.dm [hunk 1]
```diff
@@ -252,4 +252,4 @@
...
+	info = "If you find this... you don't need to know who I am.<BR><BR>You need to leave this place. I dunno what shit they did to me out here, but I don't think I'm going to be making it out of here.<BR><BR>This place... it wears down your psyche. The other researchers out here laughed it off but... They were the first to go.<BR><BR>One by one they started turning on each other. The more they found out, the more they started fighting and arguing...<BR>As I speak now, I had to... I wound up having to put most of my men down. I know what I had to do, and I know there's no way left for me to live with myself.<BR> If anyone ever finds this, just don't touch the graves.<BR><BR>DO NOT. TOUCH. THE GRAVES. Don't be a dumbass, like we all were."
```
Comment (Fikou) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:255]: you added a newline here
```suggestion
	info = "If you find this... you don't need to know who I am.<BR><BR>You need to leave this place. I dunno what shit they did to me out here, but I don't think I'm going to be making it out of here.<BR><BR>This place... it wears down your psyche. The other researchers out here laughed it off but... They were the first to go.<BR><BR>One by one they started turning on each other. The more they found out, the more they started fighting and arguing...<BR>As I speak now, I had to... I wound up having to put most of my men down. I know what I had to do, and I know there's no way left for me to live with myself.<BR> If anyone ever finds this, just don't touch the graves.<BR><BR>DO NOT. TOUCH. THE GRAVES. Don't be a dumbass, like we all were."

```

Comment (ishitbyabullet) [reply code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:255]: alright

File: code/modules/ruins/lavalandruin_code/elephantgraveyard.dm [hunk 2]
```diff
@@ -110,7 +110,7 @@
 	new /obj/effect/decal/cleanable/oil(loc)
 
 //***Grave mounds.
-/obj/structure/closet/crate/grave
+/obj/structure/closet/crate/grave // has no items inside unless you use the filled subtype
```
Comment (Mothblocks) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:113]:
```suggestion
/// has no items inside unless you use the filled subtype
/obj/structure/closet/crate/grave
```

Doc comment.

Comment (ishitbyabullet) [reply code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:113]: Alright.

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**